modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1468,34 +1857,30 @@ Type::
 | `logTypes`
 | `string`
 | `logTypes` defines the desired record types to generate. Possible values are: +
- - `FLOWS` (default) to export regular network flows +
- - `CONVERSATIONS` to generate events for started conversations, ended conversations as well as periodic "tick" updates +
- - `ENDED_CONVERSATIONS` to generate only ended conversations events +
- - `ALL` to generate both network flows and all conversations events +
+ - `Flows` (default) to export regular network flows +
+ - `Conversations` to generate events for started conversations, ended conversations as well as periodic "tick" updates +
+ - `EndedConversations` to generate only ended conversations events +
+ - `All` to generate both network flows and all conversations events +
 
 
 | `metrics`
 | `object`
 | `Metrics` define the processor configuration regarding metrics
 
-| `port`
-| `integer`
-| Port of the flow collector (host port). By convention, some values are forbidden. It must be greater than 1024 and different from 4500, 4789 and 6081.
-
-| `profilePort`
-| `integer`
-| `profilePort` allows setting up a Go pprof profiler listening to this port
+| `multiClusterDeployment`
+| `boolean`
+| Set `multiClusterDeployment` to `true` to enable multi clusters feature. This adds `clusterName` label to flows data
 
 | `resources`
 | `object`
 | `resources` are the compute resources required by this container. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
 
 |===
-== .spec.processor.debug
+== .spec.processor.advanced
 Description::
 +
 --
-`debug` allows setting some aspects of the internal configuration of the flow processor. This section is aimed exclusively for debugging and fine-grained performance optimizations, such as GOGC and GOMAXPROCS env vars. Users setting its values do it at their own risk.
+`advanced` allows setting some aspects of the internal configuration of the flow processor. This section is aimed mostly for debugging and fine-grained performance optimizations, such as `GOGC` and `GOMAXPROCS` env vars. Users setting its values do it at their own risk.
```

Comment:
```suggestion
`advanced` allows setting some aspects of the internal configuration of the flow processor. This section is aimed mostly for debugging and fine-grained performance optimizations, such as `GOGC` and `GOMAXPROCS` env vars. Set these values at your own risk.
```

Reply:
@skrthomas does this change look good to you? If so I'll take it upstream

Reply:
Yep, these suggestions look good to me.